.travis.yml

```diff
@@ -49,9 +49,9 @@ install: |
       export PATH=$PATH:$(pwd)/jsshell;
       unset ESHOST_SKIP_JSSHELL;
     elif [[ "$ESHOST_TARGET" == "chakra" ]]; then
-      wget https://aka.ms/chakracore/cc_linux_x64_1_5_2;
-      tar -xvf cc_linux_x64_1_5_2;
-      export PATH=$PATH:$(pwd)/ChakraCoreFiles/bin;
+      wget https://s3.amazonaws.com/eshost-builds/ChakraCore-Debug_1_7_0.tar.gz;
+      tar -xvf ChakraCore-Debug_1_7_0.tar.gz;
+      export PATH=$PATH:$(pwd)/out/Debug;
```

Comment:
I'm going to make one last change here, to put the tarball in a different remote directory.